src/components/header/_header.scss

```diff
@@ -30,37 +43,94 @@ $govuk-header-link-underline-thickness: 3px;
     &:focus {
       color: #0b0c0c;
     }
+
+    .govuk-template--rebranded & {
+      @include govuk-link-common; // explicit choice to differ from govuk-frontend header links
+      @include govuk-link-style-inverse;
+
+      font-weight: normal;
+
+      &:hover {
+        text-decoration-thickness: $govuk-header-link-underline-thickness;
+      }
+    }
   }
 
   .hmrc-language-select {
     padding-right: govuk-spacing(3);
     float: right;
 
+    .govuk-template--rebranded & {
+      padding-right: 0;
+    }
+
     .govuk-list {
       margin-bottom: 0;
 
+      .govuk-template--rebranded & {
+        @include govuk-media-query($from: desktop) {
+          padding-top: govuk-spacing(3) + 3px; // makes the baseline match gov.uk logo and service name
+        }
+      }
+
+      .govuk-header__link {
+        .govuk-template--rebranded & {
+          @include govuk-link-common; // explicit choice to differ from govuk-frontend header links
+          @include govuk-link-style-inverse;
+
+          font-weight: normal;
+
+          &:hover {
+            text-decoration-thickness: $govuk-header-link-underline-thickness;
+          }
+        }
+      }
+
       a:not(.govuk-header__link) {
         @include govuk-font($size: 16, $weight: "bold");
 
         color: govuk-colour("mid-grey");
         text-decoration: none;
+
+        &:link,
+        &:hover,
+        &:visited,
+        &:focus {
+          .govuk-template--rebranded & {
+            font-weight: normal;
+            color: #fff;
+          }
+        }
       }
     }
 
     .govuk-header__navigation-item {
       display: inline-block;
       margin-right: 0;
       border-bottom: 0;
+
+      .govuk-template--rebranded & {
+        padding-top: 0;
+        padding-bottom: govuk-spacing(2);
+      }
     }
 
     .govuk-header__navigation-item + .govuk-header__navigation-item::before {
       content: "|";
       padding-right: 2px;
       color: govuk-colour("dark-grey");
+
+      .govuk-template--rebranded & {
+        color: #ffffff50; // is this acceptable? mimics the existing light grey...
+      }
     }
   }
 }
 
+.govuk-template--rebranded .hmrc-header.hmrc-header--with-additional-navigation .govuk-header__navigation + .hmrc-language-select .govuk-list {
+  padding-top: 0;
+}
+
```

Comment:
this appears to only be used (and maybe only as a potentially configurable option, so may not actually be used in practice) in gforms

there are issues with the current language select implementation in the header in terms of accessibility

teams should not use it, but somewhat we're saying "defer because service nav being introduced to header will clear this up"